### DISCUSSION TITLE
Import old data

### DISCUSSION THREAD

**Discussion by Ganoid** [jacobwojoski/dice-stats#148] [Q&A] [answered]
Hey, is there a way to import data from the chat before I installed the module? I'd love to get a bunch of data from my year long campaign into the data.

**Comment by jacobwojoski** [comment, marked as answer]
Currently no but it is on the backlog.  Long term data always leads to the average so the idea for the module was to get session values as that’s what I found more interesting.

**Comment by Ganoid** [comment]
Thanks for the quick answer, amazing module!
